app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/DisplayImageSetFragment.kt

```diff
@@ -47,6 +52,24 @@ class DisplayImageSetFragment : Fragment() {
     private var _binding: FragmentDisplayImageSetBinding? = null
     private val binding get() = _binding!!
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        runBlocking {
```

Comment:
might make sense to put that code in the DI since it gets repeated a lot

Reply:
Good idea thanks

Reply:
Actually I am not sure that it is a good idea since injection is only called on creation and not on resumption of fragments

Reply:
depends what scope you put it in doesnt it?

Reply:
Yes, but I don't have not found such a scope